packages/spacecat-shared-data-access/test/it/db.test.js

```diff
@@ -134,22 +138,28 @@ describe('DynamoDB Integration Test', async () => {
     process.env.AWS_ACCESS_KEY_ID = 'dummy';
     process.env.AWS_SECRET_ACCESS_KEY = 'dummy';
 
-    dynamoDbLocalProcess = DynamoDbLocal.spawn({
+    dynamoDbLocalProcess = spawn({
+      detached: true,
```

Comment:
what would be the benefit of running dynamodb as a detached process in IT tests?

Reply:
i am following the recommendation from documentation here: https://github.com/chrisguttandin/dynamo-db-local?tab=readme-ov-file#detached